### PR TITLE
Update when arrive in uk Question

### PR DESCRIPTION
**Pull request by MIslam92**
### What is the context of this PR?
Now pipes persons name into question, added validation on date that is entered and updated corresponding functional/integration tests

Card: https://trello.com/c/Uz9U8m57/1485-update-when-arrive-in-uk-s-m

### How to review 
Open SR and get to that question and enter a invalid date to check if validation works.
Run tests as well.

### CODE REVIEW

File: data/en/census_household.json [hunk 1]
```diff
@@ -1646,7 +1646,7 @@
                     "id": "arrive-in-uk",
                     "questions": [{
                         "id": "arrive-in-uk-question",
-                        "title": "If you were not born in the United Kingdom, when did you most recently arrive to live here?",
+                        "title": "When did {{[answers.first_name[group_instance], answers.last_name[group_instance]] | format_household_name_possessive }} most recently arrive to live in the United Kingdom?",
```
Comment (jonnyshaw89) [inline data/en/census_household.json]: I don't think this should be possessive?

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Comment by jonnyshaw89** [comment]
@MIslam92 is this PR title wrong?

**Comment by jonnyshaw89** [comment]
@MIslam92 Is the wrong Trello card attached?

**Comment by MIslam92** [comment]
@jonnyshaw89 potentially 🙃 but amended now

File: data/en/census_household.json [hunk 2]
```diff
@@ -1645,7 +1645,7 @@
                     "id": "arrive-in-uk",
                     "questions": [{
                         "id": "arrive-in-uk-question",
-                        "title": "If you were not born in the United Kingdom, when did you most recently arrive to live here?",
+                        "title": "When did <em>{{[answers.first_name[group_instance], answers.last_name[group_instance]] | format_household_name}}</em> most recently arrive to live in the United Kingdom?",
                         "guidance": {
                             "content": [{
                                 "description": "Exclude short visits away from the UK"
```
Comment (jonnyshaw89) [inline data/en/census_household.json]: The guidance structure doesn't match the design

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Comment by jonnyshaw89** [comment]
@MIslam92 is this "in-progress"?

File: data/en/census_household.json [hunk 3]
```diff
@@ -1645,10 +1645,10 @@
                     "id": "arrive-in-uk",
                     "questions": [{
                         "id": "arrive-in-uk-question",
-                        "title": "If you were not born in the United Kingdom, when did you most recently arrive to live here?",
+                        "title": "When did <em>{{[answers.first_name[group_instance], answers.last_name[group_instance]] | format_household_name}}</em> most recently arrive to live in the United Kingdom?",
                         "guidance": {
                             "content": [{
-                                "description": "Exclude short visits away from the UK"
+                                "description": "<strong>Exclude</strong> <br /> <br />short visits away from the UK"
```
Comment (jonnyshaw89) [inline data/en/census_household.json:1651]: Pretty sure this isn't how we do this.

Comment (jonnyshaw89) [reply data/en/census_household.json:1651]: It should be more like this.
```
{
    "description": "Exclude",
    "list": [
        "Short visits away from the UK"
    ]
}
```

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**